src/Database/Dialect/TupleComparisonTranslatorTrait.php

```diff
@@ -81,8 +81,8 @@ protected function _transformTupleComparison(TupleComparison $expression, $query
         $conditions = ['OR' => []];
         foreach ($value as $tuple) {
             $item = [];
-            foreach (array_values($tuple) as $i => $value) {
-                $item[] = [$fields[$i] => $value];
+            foreach (array_values($tuple) as $i => $value2) {
```

Comment:
I am not sure if the reuse of `$value` here was a mistake or intentional.

Reply:
Probably a mistake